CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 See full changelog for the OpenAPI schema (OAS) [here](https://github.com/plaid/plaid-openapi/blob/master/CHANGELOG.md).
 
 # 9.8.0
-- Updating to OAS 2020-09-14_1.46.1
+- Updating to OAS 2020-09-14_1.54.2
```

Comment:
Same note as the others - curious why we're not bumping the client library version here